kpi/serializers/v2/service_usage.py

```diff
@@ -141,14 +141,15 @@ def _get_per_asset_usage(self, user):
 
         # Only use fields we need to improve SQL query speed
         user_assets = (
-            Asset.objects.only(
+            Asset.all_objects.only(
                 'pk',
                 'uid',
                 '_deployment_data',
                 'owner_id',
                 'name',
             )
             .select_related('owner')
+            .exclude(_deployment_data=None)
             .filter(
                 owner__in=self._users,
                 asset_type=ASSET_TYPE_SURVEY,
```

Comment:
`user_assets` seems to be used no where else than to be used to init `asset_list` 
I would probably get rid of `user_assets` and assign directly `asset_list` with the results of queryset with `values_list()`

Moreover, the query could simplified with the new `date_deployed` field. Only assets of type `'survey'` are deployed. We just need to check whether the value of the field is not null to get all deployed projects.

So what about this? 

```python 
        asset_list = list(
            Asset.all_objects.values_list('uid', flat=True).filter(
                owner__in=self._users,
                date_deployed__isnull=False,
            )
        )
        
        xforms = KobocatXForm.objects.only('bytes_sum', 'id', 'kpi_asset_uid').filter(
            kpi_asset_uid__in=asset_list,
        )
```